apps/docs/src/app/core/component-docs/card/examples/bar-chart-list-card/card-bar.component.scss

```diff
@@ -22,4 +22,15 @@
             }
         }
     }
+
+    .change-value {
```

Comment:
Please stop adding random classes with random names. There are thousands of documentation examples with custom css from where you can see how we name the classes

Reply:
corrected name now.